src/custom-elements/docs/docs.json

```
@@ -6076,6 +6080,22 @@
           "optional": false,
           "required": false
         },
+        {
+          "name": "helperMessage",
+          "type": "string",
+          "mutable": false,
+          "attr": "helper-message",
+          "reflectToAttr": true,
+          "docs": "To display an additional helper text message below the Text input",
```

We need to change the description in the Textarea component

```suggestion
          "docs": "To display an additional helper text message below the Texarea",
```